apps/comments/comments.py

```diff
@@ -42,6 +42,36 @@ class CommentsResource(Resource):
     privileges = {'POST': 'archive', 'DELETE': 'archive'}
 
 
+def encode_keys(doc, field):
+    """Encode field keys before storing.
+
+    Mongo doesn't allow `.` in dict key.
+
+    :param doc: doc
+    :param field: doc field
+    """
+    if not doc.get(field):
+        return
+    encoded = {}
+    for key, val in doc.get(field).items():
+        encoded[key.replace('.', ';')] = val
+    doc[field] = encoded
+
+
+def decode_keys(doc, field):
```

Comment:
base64 sounds good, but it makes those names very cryptic, will leave it with replace for now, we don't allow `;` in username